### PR TITLE
MM-1789 update readme with basic config settings setup information

### PR DESCRIPTION
**Pull request by jwilander**
@it33 please review this since it's a README change

### CODE REVIEW

**Comment by crspeller** [comment]
Looks good to me.

**Comment by it33** [comment]
@jwilander okay to merge. There's some super minor wording to adjust, and I'll submit a separate PR that doesn't need to be in 0.6.0. My fault for not making the documentation guidelines more clear.

File: README.md [hunk 1]
```diff
@@ -113,10 +113,20 @@ AWS Elastic Beanstalk Setup (Docker)
...
 	15. Try it out by entering the domain of the form \*.elasticbeanstalk.com found at the top of the dashboard into your browser. You can also map your own domain if you wish. 
 
-Contributing 
------------- 
- 
-To contribute to this open source project please review the Mattermost Contribution Guidelines at http://www.mattermost.org/contribute-to-mattermost/. 
+Configuration Settings
+----------------------
+
+There are a few configuration settings you might want to fiddle with when setting up your instance of Mattermost. You can edit them in ./config/config.json or ./config/config/config_docker.json if you're running a docker instance.
```
Comment (coreyhulen) [inline README.md]: fiddle

**Comment by coreyhulen** [comment]
+1

**Pull request merged**